src/components/Button/Button.tsx

```diff
@@ -48,20 +49,22 @@ export const Button = ({
   wrapper,
   isDisabled = false,
   hasDropDown,
+  hasBorder,
```

Comment:
I don't think we should have this options available for all button types. Instead we should consider updating Ghost to have a couple colour options... Maybe update to have Ghost Red, and Ghost Dark Grey, etc... Or a conditional option to set from a list of colour options when ghost is being used... Open to options

Reply:
Ok let's ignore this change as you are now using the light grey button variant.